src/app/sandbox-webapp/sandbox-webapp.component.html

```diff
@@ -63,6 +63,9 @@
 
                   <button (click)="rotate('LEFT')">rotate left</button>
                   <button (click)="rotate('RIGHT')">rotate right</button>
+                  <button (click)="rotateSinglePage('LEFT', 0)">rotate 1st page left</button>
+                  <button (click)="rotateSinglePage('RIGHT', 0)">rotate 1st page right</button>
```

Comment:
is there a way to get the current page (based on coordinates maybe?), so that this can be used for all pages, not just the first one?